flight/UAVObjects/inc/uavobjectmanager.h

```diff
@@ -92,7 +92,7 @@ typedef enum {
 	EV_UPDATED_MANUAL = 0x04, /** Object update event manually generated */
 	EV_UPDATED_PERIODIC = 0x08, /** Object update from periodic event */
 	EV_UPDATE_REQ = 0x10, /** Request to update object data */
-	EV_UPDATED_THROTTLED_DIRTY = 0x80 /** Indicates a throttled object has been updated but not sent **/
+	EV_UPDATED_THROTTLED_DIRTY = 0x20 /** Indicates a throttled object has been updated but not sent **/
```

Comment:
I think the reason we chose this to be ```0x80``` was to give space for other commands to grow in the space, and leave the "state" flag as the MSB. I have no objections to it moving, though.